lutris/runners/winesteam.py

```diff
@@ -300,30 +301,34 @@ def get_default_steamapps_path(self):
         if steamapps_paths:
             return steamapps_paths[0]
 
-    def create_prefix(self, prefix_dir):
+    def create_prefix(self, prefix_dir, arch=None):
         logger.debug("Creating default winesteam prefix")
         wine_path = self.get_executable()
 
         if not os.path.exists(os.path.dirname(prefix_dir)):
             os.makedirs(os.path.dirname(prefix_dir))
-        create_prefix(prefix_dir, arch=self.wine_arch, wine_path=wine_path)
+        create_prefix(prefix_dir, arch=arch, wine_path=wine_path)
 
         # Fix steam text display
+        # Since Wine 1.7.50, DirectWrite has been fixed.
         set_regedit("HKEY_CURRENT_USER\Software\Valve\Steam",
-                    'DWriteEnable', '0', 'REG_DWORD',
+                    'DWriteEnable', '1', 'REG_DWORD',
                     wine_path=self.get_executable(),
                     prefix=prefix_dir)
 
-    def get_default_prefix(self):
+    def get_default_prefix(self, arch=None):
```

Comment:
Maybe explicitly set `'win32'` as the default (like for `get_or_create_default_prefix()`)? 64-bit is the default for more and more things, so explicitly declaring the current default here and handling both/all cases explicitly in the code would mean that whenever Lutris decides to switch to 64-bit Wine by default, it'll only have to update this line. (Though the argument is transferred over straight from `get_or_create_default_prefix()`, so this would also mean maintained the default both places…)